src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserControllerTest.kt

```diff
@@ -117,6 +117,45 @@ class UserControllerTest {
     }
   }
 
+  @Nested
+  inner class CreateLinkedGeneralUser {
+    @Test
+    fun `create Linked General user`() {
+      val generalCaseLoads = listOf(
+        PrisonCaseload("NWEB", "Nomis-web Application"),
+        PrisonCaseload("BXI", "Brixton (HMP)"),
+      )
+      val adminCaseLoads = listOf(
+        PrisonCaseload("NWEB", "Nomis-web Application"),
+        PrisonCaseload("CADM_I", "Central Administration Caseload For Hmps"),
+      )
+
+      val generalAccount = UserCaseload(
+        "TEST_USER_GEN",
+        false,
+        PrisonUsageType.GENERAL,
+        generalCaseLoads.get(1),
+        generalCaseLoads,
+      )
+      val adminAccount =
+        UserCaseload("TEST_USER_ADM", false, PrisonUsageType.ADMIN, adminCaseLoads.get(1), adminCaseLoads)
+      val createLinkedGeneralUserRequest = CreateLinkedGeneralUserRequest("TEST_USER_ADM", "TEST_USER_GEN", "BXI")
+      whenever(userService.createLinkedGeneralUser(createLinkedGeneralUserRequest)).thenReturn(
+        PrisonStaffUser(
+          100,
+          "First",
+          "Last",
+          "ACTIVE",
+          "f.l@justice.gov.uk",
+          generalAccount,
+          adminAccount,
+        ),
+      )
+      userController.createLinkedGeneralUser(createLinkedGeneralUserRequest)
+      verify(userService).createLinkedGeneralUser(createLinkedGeneralUserRequest)
```

Comment:
This test verifies that user service is called, which is fine, but does not verify that the model returned by the service layer is correctly converted and returned. We need that test somewhere, either in the integration test or in this unit test. It should be fully covered by the integration test once my earlier comment is addressed. It would be useful if this test asserted additionally that a not null response is received from the controller. You may also be able to slim down the amount of test data set up in this test given that it is not really used here?